Example/Tests/CommandTests.swift

```diff
@@ -0,0 +1,43 @@
+// Copyright DApps Platform Inc. All rights reserved.
```

Comment:
should we use "Trust Wallet"? not DApps Platform

Reply:
This is the header I see in all iOS files. When you asked for the latest header I really don't know what you mean.